utils_cv/tracking/model.py

```diff
@@ -236,7 +236,80 @@ def save(self, path) -> None:
 
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
+                        
+    def savetxt_results(self,
+                        results: Dict[int, List[TrackingBbox]],
+                        exp_name: str = 'results',
+                        root_path: str = None,
+                        result_filename: str = 'results.txt',
+                        if_tmp: bool = True) -> str:
+        """Save tracking results to txt in tmp directory or provided path.
+        
+        Args:
+            results: prediction results from predict() function, i.e. Dict[int, List[TrackingBbox]]
+            exp_name: subfolder for each experiment
+            root_path: results saved root path. Default: None
+            result_filename: saved prediction results txt file; End with '.txt'
+            if_tmp: if saved in tmp directory. If set it False, root_path has to be provided.
+        Returns:
+            result_file_path: saved prediction results txt file path
+        """
+        if if_tmp or not root_path:
+            with tempfile.TemporaryDirectory() as tmpdir1:
+                os.makedirs(osp.join(tmpdir1, exp_name))
+                result_file_path = osp.join(tmpdir1, exp_name, result_filename)
+        else:
+            result_file_path = osp.join(root_path, exp_name, result_filename)
 
+        # Save results in MOT format for evaluation            
+        bboxes_mot = boxes_to_mot(results)
+        np.savetxt(result_file_path, bboxes_mot, delimiter=",", fmt="%s")
+        return result_file_path
+
+    def evaluate_mot(self,
```

Comment:
savetxt_results, evaluate_mot and mot_accumulator don't use "self" and hence should not be member functions of the class and instead could be regular functions